### PR TITLE
Cleaning up Strategic clarity and adding more on documentations

### PR DESCRIPTION
**Pull request by digidigo**
Scott's request was to add language about the documentation to Strategic Clarity. 

I did that and cleaned up some of the words. 

But... is it better for people?  More clear?

### CODE REVIEW

File: data/expectations.js [hunk 1]
```diff
@@ -138,10 +141,12 @@ export const ROLE_EXPECTATIONS = {
...
     STRATEGIC_CLARITY: {
       behaviors: [
-        'Helps guide product and engineering toward fulfilling Strategy and Design principles. Creates documentation that connects strategy, purpose and design principles with overall team objectives.',
+        'Helps align product and engineering with Gloo Strategy and Design principles.',
         'Identifies appropriate metrics that tie back to business objectives and measures those when making decisions.',
         'Understands various product offerings and how they work together.',
-        'Integrates business needs into project planning.',
+        'Actively works toward balancing team, engineering and Gloo objectives',
+        'Discovers need for and creates and maintains architectural, process and strategic documentation for the team',
+        'Discovers , advocates for and drives small to moderately complex process changes to team process',
```
Comment (bgrayless) [inline data/expectations.js]: Extra space before the comma.
```suggestion
        'Discovers, advocates for and drives small to moderately complex process changes to team process',
```

File: data/expectations.js [hunk 2]
```diff
@@ -178,10 +183,12 @@ export const ROLE_EXPECTATIONS = {
     },
     STRATEGIC_CLARITY: {
       behaviors: [
-        'Adds clarity and nuanced understanding to Strategy and Design principles.  Ensure that we are finding effective ways to measure our success against strategy. ',
-        'Can be a representative for engineering in company-level discussions.',
+        'Adds clarity and nuanced understanding to Strategy and Design principles.',
+        'Represents engineering in cross org discussions.',
         'Helps guide their team on broader impact and mentors them on how their decisions impact Gloo.',
         'Aligns technology strategy with product strategy in partnership with Directors and VP.',
+        'Leads team to create and maintain architectural , process and strategic documentation for the engineering organization',
```
Comment (bgrayless) [inline data/expectations.js]:
```suggestion
        'Leads team to create and maintain architectural, process and strategic documentation for the engineering organization',
```

File: data/expectations.js [hunk 3]
```diff
@@ -178,10 +183,12 @@ export const ROLE_EXPECTATIONS = {
     },
     STRATEGIC_CLARITY: {
       behaviors: [
-        'Adds clarity and nuanced understanding to Strategy and Design principles.  Ensure that we are finding effective ways to measure our success against strategy. ',
-        'Can be a representative for engineering in company-level discussions.',
+        'Adds clarity and nuanced understanding to Strategy and Design principles.',
+        'Represents engineering in cross org discussions.',
```
Comment (bgrayless) [inline data/expectations.js]:
```suggestion
        'Represents engineering in cross-org discussions.',
```

File: data/expectations.js [hunk 4]
```diff
@@ -251,8 +258,11 @@ export const ROLE_EXPECTATIONS = {
     STRATEGIC_CLARITY: {
       behaviors: [
         'Understands, promotes and clarifies the company vision and principles throughout engineering.',
+        'Represents engineering to outside partners and stakeholders.',
         'Connects the work we’re doing to the principles and goals of the company.',
-        'Demonstrates a deep understanding of entire product feature set.',
+        'Demonstrates a deep understanding future product roadmap.',
+        'Leads the Gloo organization in creating and maintenance of architectural , process and strategic documentation.',
```
Comment (bgrayless) [inline data/expectations.js]:
```suggestion
        'Leads the Gloo organization in creating and maintenance of architectural, process and strategic documentation.',
```

File: data/expectations.js [hunk 5]
```diff
@@ -251,8 +258,11 @@ export const ROLE_EXPECTATIONS = {
...
+        'Discovers, advocates for and successfully improves process or strategy across Gloo organization',
```
Comment (bgrayless) [inline data/expectations.js]:
```suggestion
        'Discovers, advocates for, and successfully improves process or strategy across Gloo organization',
```

**Review by bgrayless — DISMISSED** [review]

**Comment by goldenshun** [comment]
Please update `skills.js` description for Strategic Clarity.

**Comment by digidigo** [comment]
@bgrayless @goldenshun please re-review so that I can take this off my list.

**Review by seanconnollydev — APPROVED** [review]

**Pull request merged**